bundles/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastEventReceiver.java

```diff
@@ -56,6 +56,8 @@ public void connectionEventReceived(final @NonNullByDefault({}) ChromeCastConnec
 
     @Override
     public void spontaneousEventReceived(final @NonNullByDefault({}) ChromeCastSpontaneousEvent event) {
+        logger.trace("Received an {} event (class={})", event.getType(), event.getData());
```

Comment:
Just a note: I see below a similar log (case for unknow) which uses `event.getType().getDataClass()` and not `event.getData()` for class

Reply:
Yes, this is intentionally.